tcos: fix indentation

jmastr opened the pull request:
Increase readability. No content-based changes.

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
- [ ] Documentation is added or updated
- [x] PKCS#11 module is tested
- [ ] Windows minidriver is tested
- [ ] macOS tokend is tested

Jakuje commented on src/libopensc/card-tcos.c:
I tried to find the last discussion about coding style, but I was not successful. The only thing I found was some ancient wiki referencing linux kernel coding style, which allows only `} else {` on single line so I would try to stick with that as we rewrite things.

https://github.com/OpenSC/OpenSC/wiki/Development-Policy

Jakuje commented on src/libopensc/card-tcos.c:
```diff
@@ -351,28 +351,28 @@ static int tcos_select_file(sc_card_t *card,
+			pathlen = 0;
+			break;
+		default:
+			SC_FUNC_RETURN(ctx, SC_LOG_DEBUG_VERBOSE, SC_ERROR_INVALID_ARGUMENTS);
```
The same as above -- the switch and case should be on the same indentation level according to kernel coding style:

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/coding-style.rst#n47

Jakuje commented on src/libopensc/card-tcos.c:
```diff
@@ -436,9 +436,9 @@ static int tcos_list_files(sc_card_t *card, u8 *buf, size_t buflen)
-			 apdu.resplen / 2, p1 == 1 ? "DF" : "EF");
+		sc_log(ctx,
+				"got %"SC_FORMAT_LEN_SIZE_T"u %s-FileIDs\n",
+				apdu.resplen / 2, p1 == 1 ? "DF" : "EF");
```
+1 for removing the bogus spaces, but this seems to me as over-intentation. Similar problems are in the following diffs.

Jakuje commented on src/libopensc/card-tcos.c:
```diff
@@ -698,10 +698,10 @@ static int tcos_get_serialnr(sc_card_t *card, sc_serial_number_t *serial)
+		case SC_CARDCTL_TCOS_SETPERM:
+			return tcos_setperm(card, !!ptr);
+		case SC_CARDCTL_GET_SERIALNR:
+			return tcos_get_serialnr(card, (sc_serial_number_t *)ptr);
```
Again, over-indentation as mentioned in the previous comments

jmastr commented:
Hey @Jakuje Thanks for your feedback. I incorporated your comments. Please check the diff again.

According to https://www.kernel.org/doc/html/v4.12/process/coding-style.html#placing-braces-and-spaces:

```
This does not apply if only one branch of a conditional statement is a single statement; in the latter case use braces in both branches:

if (condition) {
        do_this();
        do_that();
} else {
        otherwise();
}
```

So I added a few brackets.
